pkg/clusterresourceoverride/exempt.go

```diff
@@ -0,0 +1,27 @@
+package clusterresourceoverride
+
+import (
+	"strings"
+)
+
+// this a real shame to be special cased.
+var (
+	forbiddenNames    = []string{"openshift", "kubernetes", "kube"}
+	forbiddenPrefixes = []string{"openshift-", "kubernetes-", "kube-"}
+)
+
```

Comment:
This has been ported from 3.x as is, I am thinking of removing this and instead utilizing `namespaceSelector` of webhook configuration, as shown below
```
webhooks:
  - name: clusterresourceoverrides.admission.autoscaling.openshift.io
    namespaceSelector:
      matchExpressions:
        - key: runlevel
          operator: NotIn
          values: ["0","1"]
``` 

I inspected a 4.3 cluster, all `openshift-*` namespaces have the following label
```
    openshift.io/run-level: "1"
```

But the following namespaces do not have the above label.
* kube-node-lease
* kube-public
* kube-system

If these `kube-` namespaces are not excluded from the webhook, are there any impacts on the cluster if the webhook is not running? I assume we can ignore `kube-public` but I am not sure about `kube-system`.  @deads2k @sanchezl  any thoughts?